scripts/docker

```diff
@@ -1,14 +1,16 @@
 #!/bin/bash
 
+CUR_VER=$(./scripts/echo-version package.json)
+
 if [ "$TRAVIS_PULL_REQUEST" != "false" ]
 then
     echo info::docker:: branch is a pull request, skipping docker 
-elif [ "$BRANCH" = 'master' ]
+elif [ "$BRANCH" = 'master' ] || [ "$BRANCH" == 'new-k8s-deployment' ]
```

Comment:
is `[ "$BRANCH" == 'new-k8s-deployment' ]` is temporary code? if so can you please remove it?